Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -110,8 +137,8 @@ - (void)testInvalidPasswordAuthentication {
 
 /// A non-existsing user should not be able to log in and a corresponding error should be generated.
 - (void)testNonExistingUsernameAuthentication {
-    RLMSyncCredentials *credentials = [RLMSyncTestCase basicCredentialsWithName:NSStringFromSelector(_cmd)
-                                                                       register:NO];
+    RLMAppCredentials *credentials = [RLMAppCredentials credentialsWithUsername:@"INVALID_USERNAME"
+    password:@"INVALID_PASSWORD"];
```

Comment:
There's a bunch of places here where subsequent lines are indented incorrect or not at all.